lib/ethers/rpc_client/ethereumex_http_client.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Ethers.RpcClient.EthereumexHttpClient do
+  @behaviour Ethers.RpcClient.Adapter
+
+  @exclude_delegation [:eth_get_logs]
+
+  for {func, arity} <- Ethers.RpcClient.Adapter.behaviour_info(:callbacks),
+      func not in @exclude_delegation do
+    args = Macro.generate_arguments(arity, __MODULE__)
+    @impl true
+    def unquote(func)(unquote_splicing(args)) do
+      apply(Ethereumex.HttpClient, unquote(func), [unquote_splicing(args)])
+    end
+  end
+
+  @impl true
+  def eth_get_logs(params, opts) do
+    params
+    |> replace_key(:from_block, :fromBlock)
+    |> replace_key(:to_block, :toBlock)
```

Comment:
Isn't this inverted? The function arguments are `replace_key(map, old_key, new_key)` so first comes the old and then the new.
```suggestion
    |> replace_key(:fromBlock, :from_block)
    |> replace_key(:toBlock, :to_block)
```

Reply:
NVM, this is correct.